src/kit/shell/src/shellWindows.c

```diff
@@ -274,32 +276,34 @@ int32_t shellReadCommand(TAOS *con, char command[]) {
   // Read input.
   void *console = GetStdHandle(STD_INPUT_HANDLE);
   unsigned long read;
-  wchar_t c;
+  wchar_t c[SHELL_INPUT_MAX_COMMAND_SIZE];
```

Comment:
how about malloc()/free() instead of using an array on stack?

Reply:
ok